### PR TITLE
Add config docs for file discovery

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -150,6 +154,33 @@ names:
...
 
 
+### File based SD configurations `<file_sd_config>`
+
+File based service discovery provides a more dynamic way to configure static targets
+and serves as an interface to plug in custom service discovery mechanisms.
+
+It reads a set of files containing a list of zero or more `<target_group>`s. Changes to
+all defined files are detected via disk watches and applied immediately. Files may be
+provided in YAML or JSON format. Only changes resulting in well-formated target groups
```
Comment (juliusv) [inline content/docs/operating/configuration.md]: `well-formated` -> either `well-formatted` or `well-formed`

File: content/docs/operating/configuration.md [hunk 2]
```diff
@@ -150,6 +154,33 @@ names:
...
+are applied.
+As a fallback, the file contents are re-read in the specified refresh interval.
+
+Each target has the `__meta_filepath` label attached. Its value is set to filepath from
```
Comment (juliusv) [inline content/docs/operating/configuration.md]: "to filepath" -> "to the filepath"?

File: content/docs/operating/configuration.md (hunk 1 above)
Comment (juliusv) [inline content/docs/operating/configuration.md]: Could you provide a JSON example or specification somewhere? My guess is that most people will want to use JSON for this purpose, but won't know what it should look like...

**Comment by fabxc** [comment]
Done.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**